Mlem/App/Views/Shared/ExportableViews/ExportableCommentEditorView.swift

```diff
@@ -36,56 +51,72 @@ struct ExportableCommentEditorView: View {
+    // swiftlint:disable:next function_body_length
+    func content(data: ExportableCommentData) -> some View {
         ScrollView {
-            exportableComment
+            exportableComment(data: data)
                 .padding(.bottom, 200)
         }
         .presentationBackground(.themedGroupedBackground)
         .overlay(alignment: .bottom) {
-            ExportableViewControlOverlay { createImageFromView(exportableComment) }
+            ExportableViewControlOverlay { createImageFromView(exportableComment(data: data)) }
         }
         .toolbar {
             ToolbarItem(placement: .topBarLeading) {
                 CloseButtonView(ios18Label: .cancel)
             }
             ToolbarItem(placement: .topBarTrailing) {
                 Menu("Details", icon: .general.configure) {
-                    Toggle("Creator", icon: .lemmy.person, isOn: $showCreator)
-                    Toggle("Stats", icon: .lemmy.votes, isOn: $showStats)
+                    Section(threadLength > 1 ? "Comments" : "Comment") {
+                        Toggle("Creator", icon: .lemmy.person, isOn: $showCreator)
+                        Toggle("Stats", icon: .lemmy.votes, isOn: $showStats)
+                    }
                     
-                    if comment is any Comment2Providing {
-                        Toggle("Post", icon: .lemmy.post, isOn: $showPost)
-                        if showPost {
-                            Menu("Post Details", icon: .general.configure) {
-                                Toggle("Community", icon: .lemmy.community, isOn: $postShowCommunity)
-                                Toggle("Creator", icon: .lemmy.person, isOn: $postShowCreator)
-                                Toggle("Stats", icon: .lemmy.votes, isOn: $postShowStats)
+                    if data.numComments > 1 {
+                        ControlGroup("Parent Comments") {
+                            Button {
+                                assert(threadLength > 1, "Cannot decrease thread length below 1")
+                                threadLength -= 1
+                            } label: {
+                                Image(icon: .general.remove)
                             }
-                            .menuActionDismissBehavior(.disabled) // this doesn't work but I think that's a bug
+                            .disabled(threadLength == 1)
+                            
+                            Text("\(threadLength - 1)")
+                            
+                            Button {
+                                assert(
+                                    threadLength < min(8, data.numComments),
+                                    "Cannot increase thread length beyond \(min(8, data.numComments))"
+                                )
+                                threadLength += 1
+                            } label: {
+                                Image(icon: .general.add)
+                            }
```

Comment:
We should provide a title and hide it with `.labelStyle`, which is better for accessibility

```suggestion
                            Button("Add Comment", icon: .general.add) {
                                assert(
                                    threadLength < min(8, data.numComments),
                                    "Cannot increase thread length beyond \(min(8, data.numComments))"
                                )
                                threadLength += 1
                            }
                            .labelStyle(.iconOnly)
```